mne/viz/ica.py

```diff
@@ -327,7 +365,22 @@ def plot_ica_scores(ica, scores, exclude=None, axhline=None,
     else:
         axes = [axes]
     plt.suptitle(title)
-    for this_scores, ax in zip(scores, axes):
+
+    if labels == 'ecg':
+        labels = [l for l in ica.labels_ if l.startswith('ecg')]
+    elif labels == 'eog':
+        labels = [l for l in ica.labels_ if l.startswith('eog/')]
```

Comment:
Do we need the backslash? I assume the idea is to look for something like "eog/blinks", but for many experiments/layouts (e.g. sparse sensor coverage, short visual experiment), there is only one type of eye component.